src/seedu/addressbook/AddressBook.java

```diff
@@ -608,7 +614,7 @@ private static String getUserInput() {
         return inputLine;
     }
 
-   /*
+    /*
```

Comment:
Is this change intended? Seems wrong to make the comment gutter mis-aligned.

Reply:
Line 617 was one space short of the required indentation level. Going to fix this by aligning the other lines in the comment to match.